### DISCUSSION TITLE
Broken link to discussion forum?

### DISCUSSION THREAD

**Discussion by OmarShehata** [gpuweb/gpuweb#2192] [Ideas/Feedback]
I was trying to figure out what a good place to ask questions was, and saw a link to a "discussion forum" on this page: https://www.w3.org/community/gpu/

But to my dismay, that just linked to the GitHub repo! I almost gave up and moved on until I realized GH repos now have a "discussions" tab so I found this lovely community.

Perhaps it would be helpful especially for newcomers if that linked directly to https://github.com/gpuweb/gpuweb/discussions ?

**Comment by Kangz** [comment]
@tidoust I'm trying to edit that page but can't seem to do it. Do you have rights for that?

**Reply by tidoust** [reply]
Yes, I updated the link to https://github.com/gpuweb/gpuweb/discussions

**Reply by Kangz** [reply]
Thank you!
